speech/tests/system.py

```diff
@@ -245,3 +247,41 @@ def test_stream_recognize_single_utterance(self):
             for results in self._make_streaming_request(
                     file_obj, single_utterance=False):
                 self._check_results(results.alternatives)
+
+    def test_long_running_recognize(self):
+
```

Comment:
@chemelnucfin Please remove this empty line and just "start the code"

Reply:
ok.